app/jobs/import_needs_job.rb

```diff
@@ -1,36 +1,18 @@
-require 'google/apis/sheets_v4'
 class ImportNeedsJob < ApplicationJob
   queue_as :default
 
   def perform(*args)
-    puts "Starting ImportNeedsJob #{Time.now}"
-
-    service = Google::Apis::SheetsV4::SheetsService.new
-    service.client_options.application_name = 'Harvey Needs'
-    scopes =  ['https://www.googleapis.com/auth/drive']
-    service.authorization = Google::Auth.get_application_default(scopes)
-    spreadsheet_id = '14GHRHQ_7cqVrj0B7HCTVE5EbfpNFMbSI9Gi8azQyn-k'
-    range = 'Needs!A1:N'
-    response = service.get_spreadsheet_values(spreadsheet_id, range)
-
-    headers = response.values.shift.map(&:parameterize).map(&:underscore)
-
-    ApplicationRecord.connection.transaction do
-      Need.delete_all
-      response.values.each do |row|
-        values = Hash[headers.zip(row)]
-
-        values["are_volunteers_needed"] = values["are_volunteers_needed"] =~ /no/i ? false : true
-
-        values["are_supplies_needed"] = values["are_supplies_needed"] =~ /no/i ? false : true
-
-        Need.create! values
-      end
-      puts "ImportNeedsJob Complete - {#{response.values.count - 1}}"
-
+    puts "Starting ImportNeedsJob #{Time.now} using API"
+    needs = APIImporter.needs
+    needs.each do |need|
+      # needs and cleanPhone are derived fields
+      # updatedAt is set on save to the database
+      Need.create! need.except("needs", "cleanPhone", "updatedAt")
     end
```

Comment:
Does this account for duplicate records? Or is it meant to only be run when starting a new device env?

If only Dev, can we get comments here and in the readme to that effect? I don't want this ever accidently run in production.

Reply:
I could make it account for dups, but the issue ( #20 ) indicated for developer help.

Let me take a look and see what I can do.

Reply:
Looks like the current imports just delete all data first, then import everything, which is already pretty heavy-handed.

See:

* https://github.com/sketch-city/harvey-api/blob/master/app/jobs/import_needs_job.rb#L19
* https://github.com/sketch-city/harvey-api/blob/master/app/jobs/import_shelters_job.rb#L19

Reply:
Dev only then. Makes good sense.

Reply:
@ironcoconut this is for dev only, setup only, one time only. Basically only in a `db:seed`

Reply:
My bad. I forgot that we discussed this earlier.

Reply:
@ironcoconut @jwo Did you want me to add a `delete_all` before importing in each job, like the Google Sheets one has?

Reply:
I think it's ok as is.